apps/preview/next/pages/examples/SfScrollable.tsx

```diff
@@ -0,0 +1,107 @@
+import classNames from 'classnames';
+import { SfScrollable, SfScrollableDirection, SfScrollableButtonsPlacement } from '@storefront-ui/react';
+import { prepareControls } from '../../components/utils/Controls';
+import ComponentExample from '../../components/utils/ComponentExample';
+import { ExamplePageLayout } from '../examples';
+
+function Example() {
+  const { state, controls } = prepareControls(
+    [
+      {
+        type: 'text',
+        modelName: 'children',
+        description: 'Only for demonstration purposes. Default slot, replaces example scrollable content',
+      },
+      {
+        type: 'select',
+        modelName: 'direction',
+        options: Object.keys(SfScrollableDirection),
+        propType: 'enum',
+        propDefaultValue: SfScrollableDirection.horizontal,
+        description: 'Direction of scrollable',
+      },
+      {
+        type: 'select',
+        modelName: 'buttonsPlacement',
+        options: Object.keys(SfScrollableButtonsPlacement),
+        propType: 'SfScrollableButtonsPlacement',
+        propDefaultValue: SfScrollableButtonsPlacement.block,
+        description: 'Change position of next/previous buttons',
+      },
+      {
+        type: 'text',
+        modelName: 'as',
+        propType: 'string',
+        propDefaultValue: 'div',
+        description: 'Tag of scrollable HTML element',
+      },
+      {
+        type: 'boolean',
+        modelName: 'drag',
+        propType: 'boolean',
+        propDefaultValue: false,
+        description: 'Enables item dragging on desktop',
+      },
+      {
+        type: 'boolean',
+        modelName: 'snap',
+        propType: 'boolean',
+        propDefaultValue: false,
+        description: 'Only for demonstration purposes. Enables item snapping',
+      },
+      {
+        type: 'boolean',
+        modelName: 'hideScrollbar',
+        propType: 'boolean',
+        propDefaultValue: false,
+        description: 'Only for demonstration purposes. Hides scrollbar',
+      },
+      {
+        type: 'text',
+        modelName: 'totalItems',
+        propDefaultValue: '10',
+        description: 'Only for demonstration purposes. Total number of items',
+      },
+    ],
+    {
+      children: undefined,
+      direction: SfScrollableDirection.horizontal,
+      buttonsPlacement: SfScrollableButtonsPlacement.block,
+      as: undefined,
+      drag: undefined,
+      snap: undefined,
+      hideScrollbar: false,
+      totalItems: '20',
+    },
+  );
+
+  return (
+    <ComponentExample controls={{ state, controls }}>
+      <SfScrollable
+        drag={state.get.drag}
+        direction={state.get.direction}
+        buttonsPlacement={state.get.buttonsPlacement}
+        className={classNames('w-full items-center', {
+          'snap-x snap-mandatory': state.get.snap,
+          "[&::-webkit-scrollbar]:hidden [-ms-overflow-style:'none'] [scrollbar-width:'none']": state.get.hideScrollbar,
+        })}
+      >
+        {state.get.children ||
```

Comment:
If I put some text here instead of this nice array, it'd look a bit awkward I think. 😄 Maybe it would be better not to change children in this example.

Reply:
yep I was thinking the same, but this wasnt my code, will change this

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2571/commits/7050042f8579bdc5c765f508bdfac3b3ccb2a696